requirements/constraints.txt

```diff
@@ -139,7 +139,7 @@ cryptography==3.2.1
 Sphinx==3.3.0
 
 # incremental upgrade approach
-djangorestframework==3.11.0
+djangorestframework==3.12.0
```

Comment:
The latest release is 3.12.2; shouldn't we just remove the constraint at this point?

Reply:
Removed the constraint of `djangorestframework`.